### ISSUE TITLE
A11y: [Report a Problem] Description field lacks programmatic label and hel...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5261]
- **Checkpoint ID:** 11.1.3.1d/15.1
- **View/Component:** Report a Problem

### Description
Description field lacks programmatic label and helper text link.

### Recommendation
Associate visible labels/helper text with the text field.

--- 
*Report Date: 20.02.2026*
